### PR TITLE
add specs/ja/v1.0.0.md for Japanese translation

### PR DESCRIPTION
**Pull request by juggernautjp**
Latest version (v1.0.0) has not translated to Japanese.
So, I translated TOML v1.0.0 to Japanese and add specs/ja/v1.0.0.md.

Thank you in advance, if you merge the latest Japanese specs.

### CODE REVIEW

**Comment by cannikin** [comment]
Hello! There's a strange case where the english name of the anchor is appearing in text next to the headings:

image
This isn't present in the other translations, for example Chinese:

**Comment by TonalidadeHidrica** [comment]
I want to take a look at the issue but I cannot `yarn dev` (on MacOS and node v18).  Could you tell me how I can build the website locally?  @cannikin

**Comment by arp242** [comment]
What problem did you have running it locally @TonalidadeHidrica?

File: specs/ja/v1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,948 @@
...
+- [ローカルの時間](#local-time)
+- [配列](#array)
+- [テーブル](#table)
+- [インライン テーブル](#inline-table)
+- [テーブルの配列](#array-of-tables)
+- [ファイル名の拡張子](#filename-extension)
+- [MIME タイプ](#mime-type)
+- [ABNF 文法](#abnf-grammar)
+
+仕様 {#spec}
```
Comment (arp242) [inline specs/ja/v1.0.0.md:38]: You should just remove these `{#spec}` markers (which doesn't seem supported by the Markdown parser) and use translated links; that's what all the others do:

    - [规格](#规格)
    - [Spécifications](#spécifications)
    - [명세](#명세)
    - [Specyfikacja](#specyfikacja)
    - [Especificação](#especificação)

**Comment by TonalidadeHidrica** [comment]
@arp242 Running yarn dev, I got an error like Error: error:0308010C:digital envelope routines::unsupported.  I googled it and figured out that it is happening on webpack and can be temporarily suppressed by NODE_OPTIONS='--openssl-legacy-provider.

Now webpack succeeds, but I got 404 errors no matter what page I access.  Am I doing something wrong?

**Comment by arp242** [comment]
Yeah, some of the redirects seem broken if only the spec is translated but not the rest of the website.

I don't know why either, I never looked at this before.

But directly accessing e.g. http://localhost:8000/ja/v1.0.0.html should work.

**Comment by arp242** [comment]
I guess that's issue #35 (?)

Also: you can click "details" after "Deploy Preview ready!" in the GitHub CI, so seems you don't strictly *need* to run it locally in the first place: https://deploy-preview-68--toml-io.netlify.app/ja/v1.0.0.html

**Comment by cannikin** [comment]
@TonalidadeHidrica try a `yarn build` first, then `yarn dev`!

**Comment by TonalidadeHidrica** [comment]
@cannikin Thanks, after running `yarn build` the built page showed up by directly entering the correct URL!  But it seems that `yarn dev` still does not relfect the updates automatically and I have to run `yarn build && yanr serve` every single time I make changes...  But at least it worked.

**Comment by cannikin** [comment]
Yeah if you're changing the specs themselves you'll need to do a build, it has to compile them into the HTML for display. But making changes to the *site* pages are picked up automatically by `yarn dev`!

File: specs/ja/v1.0.0.md (hunk 1 above)
Comment (cannikin) [reply specs/ja/v1.0.0.md:38]: What do you think about this suggestion @juggernautjp ?

Comment (TonalidadeHidrica) [reply specs/ja/v1.0.0.md:38]: I think that suggestion is reasonable.  I adopted it in my PR #75 and the issue was gone, with the anchors working perfectly.

**Pull request merged**